Cargo.toml

```diff
@@ -82,7 +82,7 @@ tempfile = "3"
 stacker = "0.1.17"
 libfuzzer-sys = "0.4"
 test_bin = "0.4.0"
-criterion = "0.4.0"
+criterion = "0.5.1"
```

Comment:
Bonus points if you add "Fixes https://github.com/rosenpass/rosenpass/issues/596" to the commit message